server/sockets/helpers/index.js

```diff
@@ -215,10 +215,35 @@ const emitRoleChange = (io, cohortClients, data, event) => {
   }
 };
 
+const emitListsMetaData = io => (cohortClients, dashboardClients) => list => {
```

Comment:
I can see that this method is duplicated. We already have.
Please check if you can merge it and use existing one

`updateListOnDashboardAndCohortView` in `server/sockets/helpers/index.js` file